handbook/engineering/incidents/on_call.md

```diff
@@ -0,0 +1,92 @@
+    - Make sure you have [setup access to kubernetes](https://about.sourcegraph.com/handbook/engineering/deployments#how-to-set-up-access-to-kubernetes) and know [how to perform operations](https://about.sourcegraph.com/handbook/engineering/deployments#kubectl-cheatsheet) like: looking at logs for a service, restarting a service, getting a command shell in a running pod (e.g. to look at what is on disk).
+1.  Take steps to resolve the issue (e.g. if a disk was full, delete any data that is safe to delete to resolve the immediate issue) if you can.
+    - Don't mark pages as "resolved". Wait for the underlying alert to auto resolve.
+    - If you have unsuccessfully attempted to figure out how to resolve a page, the page hasn't auto resolved (many do), and the issue appears important (e.g. impacts users), then get help from someone else. Prefer to contact people who you know are already awake/working.
+1.  Add an entry to the [incidents log](https://docs.google.com/document/d/1dtrOHs5STJYKvyjigL1kMm6u-W0mlyRSyVxPfKIOfEw/edit?usp=sharing)
+1.  File issues for any followup work that needs to happen.
+1.  If alerts are too noisy and/or inactionable, take actions to fix or disable alerts.
+
+## Ops incidents log
+
+All significant incidents that occur on Sourcegraph.com are recorded in the [ops incidents log](https://docs.google.com/document/d/1dtrOHs5STJYKvyjigL1kMm6u-W0mlyRSyVxPfKIOfEw/edit?usp=sharing). This helps us keep track of what has happened historically, discuss follow-up work, and gives insight into what types of incidents we see.
+
+False incidents (flaky alerts, etc.) should be tracked directly in GitHub issues and do not need a log entry.
+
+## Slack channels
+
+You'll want to be in #dev-ops, #buildkite, and #opsgenie on [Slack](../../communication/team_chat.md) in particular. Most of the work you do as the on-call engineer should be discussed in #dev-ops.
+
+## Getting help
+
+In some cases you may not know how to resolve an incident on your own, or the incident may be due to something you are not familiar with. In such cases, it is your responsbility to pull in more people or teams as needed to resolve the incident.
+
+If the incident is not urgent (use your best judgement) and can be handled asynchronously over the course of a few days, file a GitHub issue and pull in teams that way by cc'ing them with e.g. `@sourcegraph/cloud` `@sourcegraph/distribution`.
+
+If the incident is urgent, start a new thread in the #dev-ops Slack channel and cc the on-call person from the relevant team. You can find this in Slack using the `/genie` command:
+
+```
+/genie whoisoncall
+```
+
+If that person is unavailable, feel free to ping the entire team `@cloud`, `@distribution`, etc. to get help.
+
+## Schedule
+
+The authoritative schedule is in OpsGenie:
+https://app.opsgenie.com/schedule#/190e2873-1e3b-4350-b67b-2e681d542970
+
+I recommend importing the schedule into your calendar.
```

Comment:
Since the "I" is going to be different and unknown, i would keep it inpersonal
```suggestion
It is recommended to import the schedule into your calendar.
```